doc/user/sql/create-index.md

```diff
@@ -68,45 +68,33 @@ _col&lowbar;ref_**...** | The columns to use as the key into the index, listed i
     consider creating another materialized view that uses `SELECT some_subset
     FROM this_view...`.
 
-### Column order
-
-The order of indexed columns does impact the underlying structure of the index,
-i.e. columns are indexed in the order in which you list them.
-
-This can be most clearly seen when joining two relations; in these cases, you
-want the relations to have join keys listed in the same order to ensure simple
-point lookups (vs. index scans).
+### Structure
 
-For example, if we are joining relations `a` and `b` on columns `order_id` and
-`customer_id`, the following indexes would perform well:
+Indexes in Materialize have the following structure for each unique row.
 
--    ```sql
-     CREATE INDEX a_o_idx ON a(order_id, customer_id);
-     CREATE INDEX b_o_idx ON b(order_id, customer_id);
-     ```
+```nofmt
+((tuple of indexed columns), (tuple of the row, i.e. stored columns))
+```
 
--   ```sql
-    CREATE INDEX a_c_idx ON a(customer_id, order_id);
-    CREATE INDEX b_c_idx ON b(customer_id, order_id);
-    ```
+Index are maintained in ascending order for the tuple of indexed columns.
```

Comment:
This is .. true but probably not in a helpful way. The indexes are ordered by the `Row` containing the key, but these are ordered by the length of allocation and then by the bytes transcribed (often the right order, but e.g. for strings again based on length).

Do you think it is important to explain the specific ordering used? I can imagine people asking, but I think the right answer is "although we do order things internally, that order isn't visible to the user" or something like that.

Reply:
Also, probably typo: "index" -> "indexes"?